Package.swift

```diff
@@ -28,8 +28,9 @@ let package = Package(
     ],
     dependencies: [
         // Dependencies declare other packages that this package depends on.
-        .package(url: "https://github.com/apple/swift-nio.git", from: "1.0.0"),
-        .package(url: "https://github.com/apple/swift-nio-ssl.git", from: "1.0.1"),
+        // TODO: Update after swift-nio and swift-nio-ssl are tagged
+        .package(url: "https://github.com/apple/swift-nio.git", .branch("master")),
```

Comment:
I think you should put `from: "2.0.0-convergence.1"` here instead of `.branch("master")`

Reply:
Thanks @weissi !